src/components/Button/index.examples.js

```diff
@@ -3,9 +3,78 @@ import {storiesOf} from '@kadira/storybook'
 import {stringFixture, clickHandlerFixture} from '../../utils/Fixtures'
 import Button from '.'
 
+const standardButtonExamples = [
+  {type: 'default'},
+  {type: 'primary'},
+  {type: 'success'},
+  {type: 'warning'},
+  {type: 'danger'}
+]
+
+const dispStyles = 'flex flex-row content-center justify-center flex-wrap items-center vh-100 bg-navy'
+
+// Standard Large Button (Default)
 storiesOf('Button')
-  .addWithInfo('API', () => (
-    <Button onClick={clickHandlerFixture}> 
-      {stringFixture}
-    </Button>
+  .addWithInfo('Large (Default)', `The Large Default Button is available in several different flavors.`, () => (
+    <div className={dispStyles}>
+      {standardButtonExamples.map((btn, i) => {
+        return (
+          <div className='mb2 mh2' key={i}>
+            <Button type={btn.type} onClick={clickHandlerFixture}>
+              {btn.type}
+            </Button>
+          </div>
+        )
+      })}
+    </div>
   ))
+
+// Standard (Small) Button
+storiesOf('Button')
+  .addWithInfo('Small', `This Small Button is available in several different flavors.`, () => (
+    <div className={dispStyles}>
+      {standardButtonExamples.map((btn, i) => {
+        return (
+          <div className='mb2 mh2' key={i}>
```

Comment:
this particular wrapper is used over and over. Maybe it is a tiny component? Not for the component framework, but just here in the storybook.